core/src/locale/pl.ts

```diff
@@ -0,0 +1,100 @@
+import type { Localization } from './types'
+
+const locale: Localization = {
+  '*': {
+    prefix: 'Każdy',
+    suffix: '',
+    text: 'Nieznany',
+    '*': {
+      empty: { text: 'każdy {{field.id}}' },
```

Comment:
Hi,

thank you for the translation!

![image](https://github.com/abichinger/vue-js-cron/assets/8122139/ddb6a1af-7827-456e-ae61-1e47a05026b6)


`{{field.id}}` only works for the English translation. Could you please replace all occurrences of `{{field.id}}` with the Polish word?